src/transformers/models/speecht5/configuration_hifigan.py

```diff
@@ -0,0 +1,103 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Hi-Fi GAN model configuration"""
+
+from transformers import PretrainedConfig
+
+
+SPEECHT5_PRETRAINED_HIFIGAN_CONFIG_ARCHIVE_MAP = {
+    "Matthijs/speecht5_hifigan": "https://huggingface.co/Matthijs/speecht5_hifigan/resolve/main/config.json",
+}
+
+
+class SpeechT5HiFiGANConfig(PretrainedConfig):
```

Comment:
I don't hink we need a separate config file for this, it can live in `configuration_speecht5.py`.